app/app.js

```diff
@@ -51,10 +56,15 @@ angular
                 controllerAs: 'catalog'
             })
             .when('/clientsManagement', {
-                templateUrl: 'components/clientsManagement/clientsManagement.html',
+                templateUrl: 'components/clientsManagement/clientsmanagement.html',
```

Comment:
please keep camelCase!!! 👎 change the file to `clientsManagement.html` next time!

😢